### DISCUSSION TITLE
Templating

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#24877] [Feature Requests]
When you install Directus fresh, your data model will be completely empty. We should allow the end user to get started with one or more predefined templates for common datamodels, like CMS / PIM / CRM / DAM etc.

RFC coming soon

**Comment by snimavat** [comment]
Template for a PIM with flexible product variants, that allows users to create any type of variants 
and ability to filter variants by selected category would be great

On discord, i see dozone of posts, all asking similar question bt eventually getting blocked
